src/figuresClasses.ts

```diff
@@ -1,11 +1,85 @@
-export interface Figure {}
+export type Color = 'red' | 'green' | 'blue' | 'yellow' | 'black' | 'white';
```

Comment:
In requirements the color should have only three possible values
<img width="601" alt="image" src="https://github.com/user-attachments/assets/7ff92ccf-ffbf-4b52-ae9b-96bb20c54df9">